apps/create-class-validation/invalid_class_long_name.py

```diff
@@ -0,0 +1,45 @@
+import weaviate
+
+client = weaviate.Client("http://localhost:8080")
+
+def invalid_long_classname(client: weaviate.Client):
+    client.schema.delete_all()
+    class_obj = {
+        "vectorizer": "none",
+        "vectorIndexConfig": {
```

Comment:
I believe it is not needed when vectorizer is turned off

Reply:
Yes. The Default vectorizer is 'none' in docker file. I can remove this line.